weave/trace/weave_init.py

```diff
@@ -178,21 +177,23 @@ def init_weave_disabled() -> InitializedClient:
     client = weave_client.WeaveClient(
         "DISABLED",
         "DISABLED",
-        init_weave_get_server("DISABLED", should_batch=False),
+        init_weave_get_server("DISABLED", "DISABLED", should_batch=False),
         ensure_project_exists=False,
     )
 
     return InitializedClient(client)
 
 
 def init_weave_get_server(
+    username: str | None = None,
```

Comment:
this is an implementation detail leak -- the server does not recognize `username`